h2o-core/src/main/java/water/rapids/Merge.java

```diff
@@ -340,7 +341,121 @@ public void map(Chunk[] cs, NewChunk[] ncs) {
     return fr;
   }
 
-  private static RadixOrder createIndex(boolean isLeft, Frame fr, int[] cols, int[][] id_maps, int[] ascending) {
+  public static Frame sortOnly(final Frame leftFrame, final int leftCols[], int[][] id_maps, int[] ascendingL) {
+
+    createIndex(true ,leftFrame,leftCols,id_maps, ascendingL);  // sort the columns.
+    System.out.print("Making BinaryMerge RPC calls ... ");
+    long t0 = System.nanoTime();
+    ArrayList<SortCombine> bmList = new ArrayList<>();
+    Futures fs = new Futures();
+
+      for (int leftMSB=0; leftMSB<=255; leftMSB++) {  // only add to RPC call if MSB is not empty
+        SingleThreadRadixOrder.OXHeader leftSortedOXHeader = DKV.getGet(getSortedOXHeaderKey(/*left=*/true, leftMSB));
```

Comment:
To perform merging, one RPC is used per MSB.  There will be 256 RPCs generated here.